src/tests/device.test.js

```diff
@@ -0,0 +1,21 @@
+import supertest from 'supertest';
+import app from '../http/app';
+
+const request = supertest(app);
+it('gets the version endpoint', async (done) => {
+  request
+    .get('/api/version')
+    .end((err, res) => {
+      expect(res.status).toBe(200);
+      done();
+    });
+});
+it('gets the verify device endpoint', async (done) => {
+  request
+    .post('/api/device/verify')
+    .send({ deviceId: 'rd-2002' })
+    .end((err, res) => {
+      expect(res.status).toBe(422);
```

Comment:
204 should be a better response status code here, I would update the api spec to reflec that.